### ISSUE TITLE
Play Intro Video when programm starts

### ISSUE THREAD

**Issue by pfedick** [pfedick/DeckerGame#78]
It should be possible to skip the intro video by pressing ESC.

Play Video and Audio from seperate files.

I found this tutorial: http://dranger.com/ffmpeg/tutorial01.html

**Comment by pfedick** [comment]
I stumpled across a few issues when using ffmpeg:

- API has changed a few times, so most tutorials are outdated and don't work.
- API is difficult to understand
- opening and finding the right codec seemed to work, but when calling "avcodec_send_packet" I always get a SIGSEGV
- HUGE amount of dependencies: when installing ffmpeg, you also get dozens of other packages for all known codecs of this world.

So I believe using ffmpeg to decode my intro video is not the right way. I think I should choose a specific codec like the free webm or h264 and use the one specific library to decode that.

**Comment by pfedick** [comment]
I found an updated and working tutorial: https://github.com/rambodrahmani/ffmpeg-video-player/tree/master/tutorial02

I also found the bug which caused the SIGSEGV. Video playback is now fully working, but not timesynced yet (as is the whole game, which depends on 60 fps).

Unfortunately I can not selectively distribute only the DLLs which I need to decode my video. Because of dependencies I have to distribute them all, which makes more than 50 unneccessary DLLs.
